src/main/java/com/ericsson/ei/jsonmerge/MergePrepare.java

```diff
@@ -206,6 +203,34 @@ public String makeJmespathArrayIndexes(String path) {
         return path;
     }
 
+    /**
+     * Example 1:
+     * 
+     * originObject: {id: eventId, type: eventType, test_cases:[{event_id:
+     * testcaseid1, test_data: testcase1data},{event_id: testcaseid2, test_data:
+     * testcase2data}]}
+     * 
+     * mergeRule: "{test_time: some_time, test_name: some_name}",
```

Comment:
I'm not sure I follow this first example. Since EI won't find the merge path based on this rule I assumed the object would be inserted at root level? Have I misunderstood this example?